aion/web/elm/src/General/View.elm

```diff
@@ -1,12 +1,17 @@
 module General.View exposing (..)
 
 import General.Models exposing (Model)
-import Html exposing (Html, a, div, i, li, p, text, ul)
-import Html.Attributes exposing (href)
+import General.Utils exposing (sliceList, colorList, defaultColor)
+import Html exposing (Html, a, div, i, li, p, text, ul, h2, button)
+import Html.Attributes exposing (href, style, class)
 import Msgs exposing (Msg)
 import Routing exposing (panelPath, roomsPath)
 import RemoteData exposing (WebData)
-import Room.Models exposing (RoomsData)
+import Room.Models exposing (RoomsData, Room)
+import Bootstrap.Grid as Grid
+import Bootstrap.Grid.Col as Col
+import Bootstrap.CDN as CDN
+import Array
```

Comment:
I would reorder imports in alphabetical way, elm-format does that automatically:
https://github.com/avh4/elm-format/blob/master/Style%20Guide/Imports.md